alibi_detect/cd/pytorch/mmd_online.py

```diff
@@ -167,30 +167,32 @@ def _configure_thresholds(self):
 
         self.thresholds = thresholds
 
-    def score(self, x_t: np.ndarray) -> Union[float, None]:
+    def _update_state(self, x_t: Union[np.ndarray, Any]):
+        self.t += 1
+        x_t = super()._preprocess_xt(x_t)
```

Comment:
While I agree that this looks pretty clean, just wanted to make sure we are happy that `_update_state` also contains steps which are not related to state, such as preprocessing of `x_t`?